packages/calcite-components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -126,6 +143,11 @@ export class RadioButtonGroup {
     }
   };
 
+  private getFocusableRadioButton(): HTMLCalciteRadioButtonElement | null {
+    const index = this.radioButtons.findIndex((radiobutton) => !radiobutton.disabled);
+    return index >= 0 ? this.radioButtons[index] : null;
```

Comment:
@anveshmekala if you're returning the button, why not just use `.find()` instead of `.findIndex`?

```ts
return this.radioButtons.find((radioButton) => !radiobutton.disabled) ?? null;
```

Reply:
my bad, i checked in the wrong commit. `.find( )` is the optimal choice here.